### ISSUE TITLE
feat: Use the sass package versus node-sass

### ISSUE THREAD

**Issue by eclectic-coding** [bridgetownrb/bridgetown#278]
## Summary
So, Libsass, which `node-sass` is a binding of, has been [deprecated](https://sass-lang.com/blog/libsass-is-deprecated) since October 20, 2020, with no new features having been added since November 2018. Therefore, according to the [Sass-lang](https://sass-lang.com/dart-sass) website:
> Dart Sass is the primary implementation of Sass, which means it gets new features before any other implementation

We might want to consider switching the default installation of Bridgetown to use `sass`. 
#273 
## Implementing
I have implemented on my Bridgetown 0.20 project and there would been to be a few tweaks to webpack, but other than that it is pretty straight forward and everything works perfectly. 

Remove `node-sass` and add these new packages as dev dependacies:
```
"fibers": "^5.0.0"
"sass": "^1.32.8",
```
Change the `sassloader` configuration in webpacker as such:
```
{
    loader: "sass-loader",
    options: {
      implementation: require("sass"),
      sassOptions: {
        fiber: false,
        includePaths: [
          path.resolve(__dirname, "src/_components")
        ],
      },
    },
  },
```
If you decide this is a direction you want to proceed, I will be glad to open a PR.

**Comment by jaredcwhite** [comment]
> I have implemented on my Bridgetown 0.20 project … I will be glad to open a PR.

Yes, please do! Thanks @eclectic-coding

**Comment by jaredcwhite** [comment]
Fixed in #279
